stablehlo/reference/Tensor.cpp

```diff
@@ -177,6 +178,12 @@ Element Tensor::get(ArrayRef<int64_t> index) const {
     }
   }
 
+  // Handle boolean type.
+  if (isSupportedBooleanType(elementType)) {
+    auto elementData = reinterpret_cast<const uint8_t *>(elementPtr);
+    return Element(elementType, *elementData == 0 ? false : true);
```

Comment:
Can be simplified to:
`return Element(elementType, *elementData? true : false);`

Reply:
Let's be really picky about the underlying representation to avoid bool <=> int mishaps:

```
if (*elementData == 0) return Element(false);
if (*elementData == 1) return Element(true);
report_fatal_error("Unsupported boolean value");
```